### DISCUSSION TITLE
Malayalam translation group

### DISCUSSION THREAD

**Discussion by Justinjdaniel** [microsoft/Web-Dev-For-Beginners#504] [Translations]
Welcome to the **Malayalam** translation group! 🎉

Created this to discuss a more accurate translation of this curriculum in Malayalam 😊
inspired by the  French Translation Group 😊

### Translations Checklist
---

- [X] Base README.md
- [X] for-teachers.md
- [ ] Quizzes
- [ ] 1-getting-started-lessons base README
   - [ ] intro-to-programming-languages README
   - [ ] intro-to-programming-languages assignment
   - [ ] github-basics README
   - [ ] accessibility README
   - [ ] accessibility assignment
- [ ] 2-js-basics base README
   - [ ] data-types README
   - [ ] data-types assignment
   - [ ] functions-methods README
   - [ ] functions-methods assignment
   - [ ] making-decisions README
   - [ ] making-decisions assignment
   - [ ] arrays-loops README
   - [ ] arrays-loops assignment
- [ ] 3-terrarium base README
   - [ ] intro-to-html README
   - [ ] intro-to-html assignment
   - [ ] intro-to-css README
   - [ ] intro-to-css assignment
   - [ ] intro-to-DOM-and-closures README
   - [ ] intro-to-DOM-and-closures assignment
- [ ] 4-typing-game base README
   - [ ] typing-game README
   - [ ] typing-game assignment
- [ ] 5-browser-extension base README
   - [ ] about-browsers README
   - [ ] about-browsers assignment
   - [ ] forms-browsers-local-storage README
   - [ ] forms-browsers-local-storage assignment
   - [ ] background-tasks-and-performance README
   - [ ] background-tasks-and-performance assignment
   - [ ] solution base README
   - [ ] start base README
- [ ] 6-space-game base README
   - [ ] introduction README
   - [ ] introduction assignment
   - [ ] drawing-to-canvas README
   - [ ] drawing-to-canvas  assignment
   - [ ] moving-elements-around README
   - [ ] moving-elements-around  assignment
   - [ ] collision-detection README
   - [ ] collision-detection assignment
   - [ ] keeping-score README
   - [ ] keeping-score assignment
   - [ ] end-condition README
   - [ ] end-condition assignment
- [ ] 7-bank-project base README
   - [ ] template-route README
   - [ ] template-route assignment
   - [ ] forms README
   - [ ] forms  assignment
   - [ ] data README
   - [ ] data assignment
   - [ ] state-management README
   - [ ] state-management  assignment
   - [X] solution base README
   - [X] API base README

**Comment by Justinjdaniel** [comment]
translations/README.ml.md
commit de9f2414032b8bc41c0254ebf31aaa9a20c56b67
![Correction](https://user-images.githubusercontent.com/62233773/139666907-7193a9b0-01b9-4946-a84e-c4c0525ece1a.jpg)

**Reply by varghesejose2020** [reply]
Hi @Justinjdaniel
